### ISSUE TITLE
"Reload samples" button not implemented in neural2d-gui.py.

### ISSUE THREAD

**Issue by davidrmiller** [davidrmiller/neural2d#2]
The Reload-samples button is intended to make it easy to switch from a training set to a validation set without having to save and reload the weights file and without having to re-invoke the GUI and the neural net. To implement that button, the neural2d-core.cpp code should free the existing SampleSet object and initialize a new one with the new sample data, and leave the program in paused mode (isRunning = false) so that the user can change other parameters as needed before resuming operation.
